### DISCUSSION TITLE
Compiled windows installer sets off antivirus

### DISCUSSION THREAD

**Discussion by jrausch12** [posit-dev/positron#3874] [Feedback]
My Avast antivirus and Comodo firewall lit up like a Christmas tree trying to get through the installer. Even on post-installation, some temp powershell scripts that run on Positron's startup get contained by the firewall. This should be addressed.

**Comment by juliasilge** [comment]
Thanks for raising this @jrausch12! We have improvements for our Windows build prioritized for the short term and will look at how to address these problems. You can see some details in #3703.
